images/dockerregistry/config.yml

```diff
@@ -27,7 +27,7 @@ middleware:
   repository:
     - name: openshift
       options:
-        acceptschema2: false
+        acceptschema2: true
```

Comment:
i needed it to test because this only supports pushing as v2.  That's temporary.